src/main/java/org/opensearch/flowframework/indices/FlowFrameworkIndicesHandler.java

```diff
@@ -395,6 +395,17 @@ public void putInitialStateToWorkflowState(String workflowId, User user, ActionL
      * @param listener action listener
      */
     public void updateTemplateInGlobalContext(String documentId, Template template, ActionListener<IndexResponse> listener) {
+        updateTemplateInGlobalContext(documentId, template, listener, false);
+    }
+
+    /**
+     * Replaces a document in the global context index
+     * @param documentId the document Id
+     * @param template the use-case template
+     * @param listener action listener
+     * @param force if set true, ignores the requirement that the provisioning is not started
+     */
+    public void updateTemplateInGlobalContext(String documentId, Template template, ActionListener<IndexResponse> listener, boolean force) {
```

Comment:
```suggestion
    public void updateTemplateInGlobalContext(String documentId, Template template, ActionListener<IndexResponse> listener, boolean isWorkflowStarted) {
```

Reply:
I'm ok with changing `force` to something else, but the point is to update it regardless of whether it's started or not.   Maybe `ignoreNotStartedCheck`?

Reply:
`ignoreNotStartedCheck` sounds better than `force`.